src/main/java/org/ibp/api/domain/ontology/AnalysisVariablesImportRequest.java

```diff
@@ -0,0 +1,53 @@
+package org.ibp.api.domain.ontology;
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.List;
+
+@AutoProperty
+public class AnalysisVariablesImportRequest {
+
```

Comment:
Can you move this to Middleware and use it as the parameter in `OntologyVariableService#createAnalysisVariables`?